manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "dxt_version": "0.1",
+  "name": "@microsoft/clarity-mcp-server",
+  "version": "1.0.5",
+  "description": "MCP Server for Microsoft Clarity based on data export API",
+  "long_description": "MCP Server for Microsoft Clarity based on data export API",
+  "author": {
+    "name": "Microsoft",
+     "url": "https://clarity.microsoft.com"
+  },
+  "icon": "icon.png",
+  "screenshots": [
+    "icon.png"
+  ],
+  "server": {
+    "type": "node",
+    "entry_point": "dist/index.js",
+    "mcp_config": {
+      "command": "node",
+      "args": [
+        "${__dirname}/dist/index.js"
+      ],
+      "env": {
+        "CLARITY_API_TOKEN": "${user_config.CLARITY_API_TOKEN}"
+      }
+    }
+  },
+  "tools_generated": true,
+  "keywords": ["api", "analytics", "productivity"],
+  "license": "MIT",
+  "compatibility": {
+    "claude_desktop": ">=0.10.0",
+    "platforms": ["darwin", "win32", "linux"],
+    "runtimes": {
+      "Node.js": ">=16.0.0"
+    }
+  },
+  "user_config": {
+    "CLARITY_API_TOKEN": {
+      "type": "string",
+      "title": "API Token",
+      "description": "Provide your API token for Microsoft Clarity",
+      "required": true,
+      "sensitive": true
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/clarity-mcp-server"
+  }
+}
```

Comment:
Please provide the missing fields: `homepage`, `documentation`, `support`, `tool`, and try to also experiment with the `prompts` and see how it will affect the DXT.